.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    "@emotion/babel-preset-css-prop",
+    "minify"
```

Comment:
let's not minify, bundlers does it

Reply:
Right, removed it in https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/62/commits/fa60e97fcd1c69a815a9e21cb440445124147db2.